test/support/TestApiConnector.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2022 The National Archives
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package support
+
+import cats.effect.IO
+
+import uk.gov.nationalarchives.omega.editorial.connectors.ApiConnector
+import uk.gov.nationalarchives.omega.editorial.models.EditSet
+import uk.gov.nationalarchives.omega.editorial.editSets
+
+object TestApiConnector extends ApiConnector(null, null, null) {
+
+  override def getEditSet(id: String): IO[EditSet] =
```

Comment:
Could we get the requested Edit Set from the `package`?

Otherwise, we're limited to just the one, in the "unit" tests.

Reply:
Sorry, I'm not sure if I understand. We currently only have one edit set in the package object. Do you think we should change the way that edit sets are represented in the package object so that we can have more than one of them, indexed by this `id`?

Reply:
Sorry - I confused myself: I was thinking of Edit Set Records, of which there are several.